src/main/scala/sbtassembly/Assembly.scala

```diff
@@ -291,9 +297,16 @@ object Assembly {
 
   private[sbtassembly] def sha1 = MessageDigest.getInstance("SHA-1")
   private[sbtassembly] def sha1content(f: File): String  = bytesToSha1String(IO.readBytes(f))
-  private[sbtassembly] def sha1name(f: File): String     = sha1string(f.getCanonicalPath)
+  private[sbtassembly] def sha1name(f: File)(implicit ao: AssemblyOption): String = sha1SafeString(f.getCanonicalPath)
```

Comment:
Would you prefer that I pass 2 parameters? I don't have a strong opinion on this subject.